resources/analysisTools/indelCallingWorkflow/checkSampleSwap_TiN.sh

```diff
@@ -30,7 +30,8 @@ ${PERL_BINARY} ${TOOL_CHECK_SAMPLE_SWAP_SCRIPT} \
     --annotate_vcf=${TOOL_ANNOTATE_VCF_FILE} \
     --gnomAD_genome=${GNOMAD_V2_1_GENOME_SNV_INDEL} \
     --gnomAD_exome=${GNOMAD_V2_1_EXOME_SNV_INDEL} \
-    --localControl_commonSNV=${LOCAL_CONTROL_PLATYPUS_SNV_INDEL} \
+    --localControl_WGS=${LOCAL_CONTROL_PLATYPUS_WGS_SNV_INDEL} \
+    --localControl_WES=${LOCAL_CONTROL_PLATYPUS_WES_SNVS_INDEL} \
```

Comment:
Likely a typo. This should be ` LOCAL_CONTROL_PLATYPUS_WES_SNV_INDEL` without the "S" in the WES `SNVS`, right?

Reply:
Ah, yes, a typo, only no-control part was tested, so missed this TiNDA typo. I have tested the TiNDA now, so it should be good.